### PR TITLE
Add learn more link cod

### LINKED ISSUE #7297: Add "Learn More" link to the Pay in Person toggle description on the card reader hub screen
This task aims to add a `Learn More` link to the "Pay in Person" toggle on the card reader hub screen. 

When clicked, it opens a chrome custom tab and takes the user to the "Cash on Delivery" section of In-Person Payments.

For Wcpay: "https://docs.woocommerce.com/document/getting-started-with-in-person-payments-with-woocommerce-payments/#add-cod-payment-method"

For Stripe: "https://docs.woocommerce.com/document/stripe/accept-in-person-payments-with-stripe/#section-8"

Screenshot 2022-08-24 at 12 20 14

### PR DESCRIPTION
**Pull request by AnirudhBhat**
Closes: #7297 

### Description

This PR adds a Learn More link to the "Pay in Person" toggle on the card reader hub screen.

When clicked, it opens a chrome custom tab and takes the user to the "Cash on Delivery" section of In-Person Payments.

For Wcpay: "https://docs.woocommerce.com/document/getting-started-with-in-person-payments-with-woocommerce-payments/#add-cod-payment-method"

For Stripe: "https://docs.woocommerce.com/document/stripe/accept-in-person-payments-with-stripe/#section-8"

#### Verifying tapping on the learn more text opens a web view
1. Navigate to the Payments screen (Menu -> Payments)
2. Tap on the "Learn more" link on the "Pay in Person" toggle option
3. Verify chrome custom tab opens IPP-related link

#### Verifying tapping on the learn more text opens a web view and takes user to the "Cash on Delivery" section
1. Navigate to the Payments screen (Menu -> Payments)
2. Tap on the "Learn more" link on the "Pay in Person" toggle option
3. Verify chrome custom tab opens IPP-related link and automatically takes user to the Cash on Delivery section.

#### Verifying tapping anywhere on the Pay in Person description text also opens a web view
1. Navigate to the Payments screen (Menu -> Payments)
2. Tap on the "Pay in Person" description text
3. Verify chrome custom tab opens IPP-related link and automatically takes user to the Cash on Delivery section.

### Images/gif

https://user-images.githubusercontent.com/1331230/187199180-e5bccd33-96fb-4ce6-ac0d-d31c15509d8f.mp4

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/cardreader/hub/CardReaderHubViewModel.kt [hunk 1]
```diff
@@ -54,13 +57,27 @@ class CardReaderHubViewModel @Inject constructor(
...
             index = 2,
             isChecked = false,
-            onToggled = { (::onCashOnDeliveryToggled)(it) }
+            onToggled = { (::onCashOnDeliveryToggled)(it) },
+            onLearnMoreClicked = ::onLearnMoreClicked
         )
     )
 
+    private fun onLearnMoreClicked() {
+        triggerEvent(
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/cardreader/hub/CardReaderHubViewModel.kt:72]: Will be tracking if this click any useful here, wdyt?

**Comment by kidinov** [comment]
While testing this change, I noticed that a user can click on the toggle multiple times, even while the previous call is still running. I'm thinking would it improve UX a bit if we will disable the toggle while the call is running? Just a thought to consider, not related to the PR

https://user-images.githubusercontent.com/4923871/187451695-6034721d-6914-48a8-9286-7d31e68fda42.mp4

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/cardreader/LearnMoreUrlProvider.kt [hunk 2]
```diff
@@ -13,15 +13,30 @@ class LearnMoreUrlProvider @Inject constructor(
...
+            LearnMoreUrl.CASH_ON_DELIVERY -> {
+                when (preferredPlugin) {
+                    STRIPE_EXTENSION_GATEWAY -> AppUrls.STRIPE_LEARN_MORE_ABOUT_PAYMENTS_CASH_ON_DELIVERY
+                    WOOCOMMERCE_PAYMENTS, null -> AppUrls.WOOCOMMERCE_LEARN_MORE_ABOUT_PAYMENTS_CASH_ON_DELIVERY
+                }
+            }
         }
     }
+
+    enum class LearnMoreUrl {
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/cardreader/LearnMoreUrlProvider.kt]: np: maybe for clarity `LearnMoreUrlType`?

**Review by kidinov — APPROVED** [review]
LGTM 👍 Left a couple of questions

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/cardreader/LearnMoreUrlProvider.kt (hunk 2 above)
Comment (AnirudhBhat) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/cardreader/LearnMoreUrlProvider.kt]: Fixed here: https://github.com/woocommerce/woocommerce-android/pull/7298/commits/9fbad1617eb1506e8d2b46d4c77c25a3971bd480

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/cardreader/hub/CardReaderHubViewModel.kt (hunk 1 above)
Comment (AnirudhBhat) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/payments/cardreader/hub/CardReaderHubViewModel.kt:72]: Good suggestion. I'm planning to add [tracking](pdfdoF-1df-p2#comment-2296) in a separate PR and will make sure to include this.

**Comment by AnirudhBhat** [comment]
>While testing this change, I noticed that a user can click on the toggle multiple times, even while the previous call is still running. I'm thinking would it improve UX a bit if we will disable the toggle while the call is running? Just a thought to consider, not related to the PR

Thanks for raising this! I thought I kinda handled it 🤷  Tweaked a little bit in this [commit](https://github.com/woocommerce/woocommerce-android/pull/7280/commits/0ef2c4ffeb26a60d2d1d2736b679a409cd735934). It's not 100% there yet but I think it's enough for this release.

**Pull request merged**